cli/commands/action_list.go

```diff
@@ -94,7 +107,7 @@ func (cmd *ActionListCommand) OnUsageError(ctx *cli.Context, err error, isSubcom
 // Action runs when no subcommands are specified
 // https://godoc.org/github.com/urfave/cli#ActionFunc
 func (cmd *ActionListCommand) Action(ctx *cli.Context) error {
-	controller, err := api.NewControllerClient(ctx)
+	controller, err := Controller(ctx)
 	if err != nil {
 		// Exit if the controller returned an error
 		return nil
```

Comment:
For this scenario, if there is an error the command won't be executed. If we return an error the urfave/cli library will exit without clear information where and why.